### ISSUE TITLE
Rename "lutro" Lua object to "love"

### ISSUE THREAD

**Issue by RobLoach** [libretro/libretro-lutro#116]
Will make porting games a lot easier.

**Comment by RobLoach** [comment]
Agreed, they are different, which makes sense for a different name. Would be good to eventually get code parity.

**Comment by rsn8887** [comment]
Why even name it differently? Either is it Love2D or it isn't.

Currently this fails to run any Love2D game. Since it is close to Love2D, it would be so much greater if it could run Love2D games. It doesn't have to run every game, but at least a few. That would be so nice.

**Comment by heuripedes** [comment]
Love2D compat would require implementing the entire `graphics` module and that is months worth of work for normal people unless one is willing to give up portability and use something like [cairo](https://cairographics.org/)/[pixman](http://www.pixman.org/) to implement all the affine transformations, blending and whatnot. I've tried both approaches and both are major PITA to make behave like Love2D while also having good performance and maintaining a similar portability level.

**Comment by rsn8887** [comment]
I am surprised about your statement since Love2d was ported to Android and IOS already and is compatible on those platforms even using shaders etc. If it was so much work, I wouldn't have thought that those ports existed.

**Comment by heuripedes** [comment]
Love2D requires certain APIs that are either subpar or not available on some platforms (some don't have shaders or flexible GPUs for instance). Lutro is available on more platforms than Love2D.

It would be probably easier to create `love2d-libretro` than make Lutro reach full Love2D compat while staying as portable as it is now.
